<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkmodeDialog</class>
 <widget class="QDialog" name="WorkmodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set L2 WorkMode</string>
  </property>
  <widget class="QGroupBox" name="gbFOV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>FOV</string>
   </property>
   <widget class="QRadioButton" name="chkWideFOV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wide</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chkStdFOV">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Std</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbPC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Point cloud</string>
   </property>
   <widget class="QRadioButton" name="chk3D">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chk2D">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2D</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbIMU">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>IMU</string>
   </property>
   <widget class="QRadioButton" name="chkIMUon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>on</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chkIMUoff">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbCOM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Communication channel</string>
   </property>
   <widget class="QRadioButton" name="chkUDP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chkSerial">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbPWR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Power on state</string>
   </property>
   <widget class="QRadioButton" name="chkPWRon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chkPWRstdby">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Standby</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnResetL2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>  Reset L2 and exit app</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetWM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Get WorkMode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetWM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set WorkMode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>chkWideFOV</tabstop>
  <tabstop>chkStdFOV</tabstop>
  <tabstop>chk3D</tabstop>
  <tabstop>chk2D</tabstop>
  <tabstop>chkIMUon</tabstop>
  <tabstop>chkIMUoff</tabstop>
  <tabstop>chkUDP</tabstop>
  <tabstop>chkSerial</tabstop>
  <tabstop>chkPWRon</tabstop>
  <tabstop>chkPWRstdby</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
